src/rad/mainframe.cpp

```diff
@@ -968,6 +968,11 @@ void MainFrame::UpdateFrame()
 	wxMenu* menuEdit = GetMenuBar()->GetMenu( GetMenuBar()->FindMenu( wxT( "Edit" ) ) );
 	wxToolBar* toolbar = GetToolBar();
 
+	toolbar->EnableTool(ID_SAVE_PRJ, AppData()->IsModified());
+	wxMenu* pmenuFile = GetMenuBar()->GetMenu(GetMenuBar()->FindMenu(_("File")));
+	if (pmenuFile)
```

Comment:
The file menu should be always there, at most i would use an assertion here to track down the _() and wxT() mismatches, but otherwise a crash if the menu is missing is ok